### PR TITLE
Fix conformance tests

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: hack/tools/tools.go [hunk 1]
```diff
@@ -25,4 +25,5 @@ import (
 	_ "sigs.k8s.io/cluster-api/hack/tools/release"          //nolint
 	_ "sigs.k8s.io/controller-tools/cmd/controller-gen"     //nolint
 	_ "sigs.k8s.io/testing_frameworks/integration"          //nolint
+	_ "sigs.k8s.io/kustomize/kustomize/v3" 					//nolint
```
Comment (joonas) [inline hack/tools/tools.go]: Alignment seems off here

Comment (vincepri) [reply hack/tools/tools.go]: Fixed, thanks

**Comment by joonas** [comment]
Step in the right direction, I like it.

/lgtm

**Comment by nader-ziada** [comment]
The conformance tests job didn't seem to pass here

**Comment by vincepri** [comment]
@nader-ziada This was to make it run, rather than pass. Need to look why it failed

**Pull request merged**